apps/next/public/apple-music-auth-native-webview/apple-music-auth-native-webview.html

```diff
@@ -1,6 +1,23 @@
 <html>
   <head>
-    <script src="https://js-cdn.music.apple.com/musickit/v3/musickit.js"></script>
+    <script>
```

Comment:
clearly we don't need this much JS, but there's no modularized apple music auth only SDK by apple 🤦